deps/exokit-bindings/canvascontext/src/canvas-context.cc

```diff
@@ -583,25 +745,50 @@ NAN_SETTER(CanvasRenderingContext2D::FontWeightSetter) {
 }
 
 NAN_GETTER(CanvasRenderingContext2D::LineHeightGetter) {
-  // nothing
+  CanvasRenderingContext2D *context = ObjectWrap::Unwrap<CanvasRenderingContext2D>(info.This());
+
+  info.GetReturnValue().Set(JS_NUM(context->lineHeight));
 }
 
 NAN_SETTER(CanvasRenderingContext2D::LineHeightSetter) {
   // Nan::HandleScope scope;
 
   if (value->IsNumber() || value->IsString()) {
     CanvasRenderingContext2D *context = ObjectWrap::Unwrap<CanvasRenderingContext2D>(info.This());
-
-    double lineHeight = TO_DOUBLE(value);
-
+    double lineHeight = 1;
+    if (value->IsString()) {
+      Nan::Utf8String text(value);
+      std::string lineHeightString(*text, text.length());
+      if (lineHeightString == "normal") {
+        lineHeight = 1;
+      } else {
+        Nan::ThrowError("lineHeight: invalid arguments");
```

Comment:
Is it correct to be throwing but still setting local `lineHeight`? I would expect either or...

Reply:
that's a mistake.